bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -11,6 +11,14 @@
 		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS"/>
 	</channel-type>
 
+	<channel-type id="ct" advanced="true">
+		<item-type>Number</item-type>
+		<label>Color Temperature</label>
+		<description>This channel supports adjusting the color temperature in Kelvin.</description>
```

Comment:
As we are adding this channel type to more and more bindings: Wouldn't it make sense to also define it as a system channel, just next to the "normal" color temperature channel type?

Reply:
Yes, absolutely. This way of controlling the color temperature has some advantages over the "normal" color temperature channel. I don't know if you followed the discussion in https://github.com/openhab/openhab-addons/issues/8260#issuecomment-761846845, the question is if we really want to continue supporting the Dimmer-based color temperature via system default channel type.

Do you think a UoM based item-type (`Number:Temperature` or new one `Number:ColorTemperature`) is conceivable? Or does it lead to misunderstandings - in general only Kelvin is valid.

Reply:
No, I hadn't followed that discussion, I've now left a comment on it.

> Do you think a UoM based item-type (Number:Temperature or new one Number:ColorTemperature) is conceivable? Or does it lead to misunderstandings - in general only Kelvin is valid.

`Number:Temperature` is imho pretty misleading. `ColorTemperature` is no existing dimension (afaik), so as the value always only makes sense in Kelvin, I guess a plain Number channel should be sufficient.

Reply:
I submitted https://github.com/openhab/openhab-core/pull/2157 to add it.